src/foam/u2/wizard/controllers/Form.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard.controllers',
+  name: 'Form',
+  extends: 'foam.u2.Controller',
+
+  issues: [
```

Comment:
!

Reply:
I'm not sure how to address this comment